src/components/list-facet/index.js

```diff
@@ -178,31 +187,41 @@ class FederatedListFacet extends React.Component {
       const listFacetHierarchyLis = [];
       // Define array of checkbox Lis which we'll populate with react fragments, per type.
       const listFacetHierarchyTermsLis = [];
+
+      /* eslint-disable react/no-array-index-key */
+
       // Iterate through types (accordion lis).
       uniqueTypes.forEach((type, i) => {
         // Populate the checkbox lis react fragments for each type.
         listFacetHierarchyTermsLis[type] = [];
-        terms[type].items.forEach((termObj, i) => termObj.facetCount
-          && listFacetHierarchyTermsLis[type].push(<li className="fs-search-accordion__content-item" key={`${termObj.term}_${termObj.facetValue}_${i}`}>
-            <label className="fs-search-accordion__checkbox-label">
-            <input
-              type="checkbox"
-              name={type}
-              className="fs-search-accordion__checkbox-input"
-              value={termObj.facetValue}
-              checked={value.indexOf(termObj.facetValue) > -1}
-              onChange={() => this.handleClick(termObj.facetValue)}
-            /> {termObj.term}
-              <span className="facet-item-amount"> ({termObj.facetCount}
-                <span className="fs-element-invisible">results</span>)
-              </span>
-            </label>
-          </li>));
+        terms[type].items.forEach((termObj, j) => termObj.facetCount
+          && listFacetHierarchyTermsLis[type].push(
+            <li className="fs-search-accordion__content-item" key={`${termObj.term}_${termObj.facetValue}_${j}`}>
+              {/* eslint jsx-a11y/label-has-associated-control: ["error", { assert: "either" } ] */}
```

Comment:
By default, the linting rule requires both nesting and `id` plus `for` attributes. However, it seemed like generating IDs at this level was discouraged, so I used an override to allow nesting the control under the label to be sufficient.